x-pack/legacy/plugins/maps/public/layers/styles/color_utils.js

```diff
@@ -64,13 +64,17 @@ export function getColorRampCenterColor(colorRampName) {
 
 // Returns an array of color stops
 // [ stop_input_1: number, stop_output_1: color, stop_input_n: number, stop_output_n: color ]
-export function getOrdinalColorRampStops(colorRampName, numberColors = GRADIENT_INTERVALS) {
+export function getOrdinalColorRampStops(colorRampName, min, max) {
   if (!colorRampName) {
     return null;
   }
+
+  const numberColors = GRADIENT_INTERVALS;
+  const delta = max - min;
   return getHexColorRangeStrings(colorRampName, numberColors).reduce(
     (accu, stopColor, idx, srcArr) => {
-      const stopNumber = idx / srcArr.length; // number between 0 and 1, increasing as index increases
+      // const stopNumber = idx / srcArr.length; // number between 0 and 1, increasing as index increases
+      const stopNumber = delta > 0 ? min + (delta * idx) / srcArr.length : 1;
```

Comment:
This is going to cause errors when there is not delta. For example, when you cluster layer only has a single object single document and there is not count range.

<img width="850" alt="Screen Shot 2020-02-25 at 3 58 33 PM" src="https://user-images.githubusercontent.com/373691/75295420-dae93580-57e7-11ea-91ae-1681ed6e6ab2.png">